content/strategy-goals/strategy/code-intelligence/index.md

```diff
@@ -106,25 +105,28 @@ Building the code graph also means we need to generate and store increased amoun
 
-Several recent efforts have shown an increasing need to unify the [code intelligence data platform](https://docs.google.com/document/d/1AjZ_d0nJVHbV75IH3jZRkrGXhsv_AXp2kS4nrw2SAQ8). These efforts show an immediate need to be able to correlate and aggregate data from different sources within an API boundary (instead of re-implemented on several clients). This will set precedent for where code goes in relation to code intelligence domains and will establish clear ownership boundaries for other product teams to build on top of. We see this engineering milestone as a large enabler for every product vertical, thus indirectly impacting usage in the mid term.
+- Make Sourcegraph a must-have for code security teams and helping react to incidents by searching and finding references to vulnerable dependency versions or errors. Having an available fully semantic graph of the code in question allows other products such as Batch Changes to provide fast fixes and turnarounds during incidents and vulnerability detections.
+- Accelerating developer onboarding by enabling the ability to search over the full transitive dependency graph.
+- Monitor code health by creating Code Insights that track deprecated dependency usages and version drift.
+- Encourage code reuse by making precise navigation the obvious way to find references to relevant usage examples.
+
+These are all currently in early stages of development and discussion. Read more about how we're thinking about the dependency graph in [RFC 593: Unlocking use cases based on the dependency graph of repositories](https://docs.google.com/document/d/1SkM8CG0IksvPEKRBRVLKipiRJTopx6Vq_hSWRJ9NyKs/edit#heading=h.trqab8y0kufp).
 
 ### How do these all tie together?
 
-Although the themes above might seem isolated or disconnected at a first glance, each one of them contributes to push forward our [Code Graph vision](../index.md). Auto-indexing serves a double purpose: it lowers the barrier to adoption by making precise code intelligence easier to configure, and also unlocks the ability to index dependencies (which play a central part in our current strategy). By adding more language support, combining it with adding package hosts support, and improving our navigation features overall we're aiming to position Sourcegraph as a superior code navigation platform that goes beyond baseline expectations and creates delightful experiences for our users.
+Every one of the themes above contributes to push forward our vision. Auto-indexing serves a double purpose: it lowers the barrier to adoption by making precise code intelligence easier to configure, and also unlocks the ability to index and navigate dependencies. By adding more language support, combined with adding package hosts support, and improving our navigation features overall we're aiming to position Sourcegraph as a superior code navigation platform that goes beyond baseline expectations and creates delightful experiences for our users.
```

Comment:
```suggestion
Auto-indexing serves a double purpose: it lowers the barrier to adoption by making precise code intelligence easier to configure and unlocks the ability to navigate dependencies. By adding more language and package host support we're aiming to position Sourcegraph as a superior code navigation platform that goes beyond baseline expectations and creates delightful experiences for our users.
```

Reply:
I'd perhaps try:

Auto-indexing serves a dual purpose:
* Lowers the barrier to adoption, meaning higher amounts of code have precise intelligence.
* Unlocks dependency navigation 

By adding precise support for more languages (and integration with their respective package host) we can provide a superior code navigation platform that pushes past the IDE paradigm (find references on all your things etc).